src/components/ComposerQuill/composer.js

```diff
@@ -4,22 +4,16 @@ import MarkdownIt from 'markdown-it';
 import Turndown from 'turndown';
 
 import Quill from './quill';
+import './styles.scss';
```

Comment:
I think I would prefer to import these in the main .scss file since we do that elsewhere. That or bring everything into the components.

Reply:
Oh, I tried this actually. There isn't a `styles/index.scss` file in this repo like there is in the web clients. I tried creating one but it didn't do anything so I must've missed something

Reply:
My bad. I actually thought I was reviewing in Cantina originally. This is fine then.